mk/linux.mk

```diff
@@ -23,9 +24,13 @@ COVERAGE=--coverage
 endif
 endif
 
+ifeq ($(HARDWARE_ARCH),x86_64)
+SIMDFLAGS=-msse4.2 -mfpmath=sse
```

Comment:
Eigen [supports](http://eigen.tuxfamily.org/index.php?title=FAQ#Which_SIMD_instruction_sets_are_supported_by_Eigen.3F) NEON instruction sets. I think we should aim to build with these.

Reply:
The page you linked says:

> On 64-bit ARM, SIMD is enabled by default, you don't have to do anything extra.

Reply:
Ok interesting. I'd only skimmed the code and I thought all the SIMD code was behind compile time flags. That's good in that case.